htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -2697,6 +2697,7 @@ public function Livraison($user, $date, $type, $comment)
 
 				$sql = "UPDATE ".$this->db->prefix()."commande_fournisseur";
 				$sql .= " SET fk_statut = ".((int) $statut);
+				$sql .= ",date_livraison = ".($date ? "'".$this->db->idate($date)."'" : 'null');
```

Comment:
The field date_livraison is the "date planed for delivery".
If i t was alreasy set, we should not erase it to not loose information.
The date of real reception is not stored for the moment (except into the list of agenda events) or if you use the module reception.
If not using this module, we could store it into a field "date_reception" in llx_commande_fournisseur but another PR to fist add the field, and then update/manage it should be done differently in 2 different PR.